pkg/controller/vm_controller.go

```diff
@@ -569,6 +585,47 @@ func (r *VMReconciler) buildVMPod(ctx context.Context, vm *virtv1alpha1.VirtualM
 					},
 				},
 			})
+
+			if iface.Vhostuser != nil {
+				type nadConfig struct {
+					Type                      string `json:"type"`
+					VhostuserSocketVolumeName string `json:"vhost_user_socket_volume_name,omitempty"`
```

Comment:
Still prefer VhostUserXXXX over Vhostuser. And since it's common to use vhost-user or vhost_user, so VhostUser may be better